es6-shim.js

```diff
@@ -1269,7 +1269,13 @@
       if (value === 0) { return value; }
       var negate = value < 0, result;
       if (negate) { value = -value; }
-      result = Math.pow(value, 1 / 3);
+      if (value === Infinity) {
```

Comment:
This looks like the code path could be hit with `-Infinity` too, is that one accurate for the `exp` + `log` path?